sdcm/utils/common.py

```diff
@@ -242,14 +244,17 @@ def download_file(self, link, dst_dir):
 
 
 def get_latest_gemini_version():
+    """search latest version of gemini tool"""
     bucket_name = 'downloads.scylladb.com'
 
     results = S3Storage(bucket_name).search_by_path(path='gemini')
     versions = set()
     for result_file in results:
-        versions.add(result_file.split('/')[1])
+        versions.add(Version(result_file.split('/')[1]))
```

Comment:
It'll be more pythonic to use list/set comprehension or generator expression instead of the loop here:

```python
versions = {Version(result_file.split("/")[1]) for result_file in results}
```

Reply:
fixed